packages/@aws-cdk/toolkit-lib/test/actions/diff.test.ts

```diff
@@ -9,6 +9,8 @@ import { Toolkit } from '../../lib/toolkit';
 import { builderFixture, disposableCloudAssemblySource, TestIoHost } from '../_helpers';
 import { MockSdk, restoreSdkMocksToDefault, setDefaultSTSMocks } from '../_helpers/mock-sdk';
 
+jest.setTimeout(10_000);
```

Comment:
Intentional?

Reply:
Yes, the test frequently fails locally for me. This is a QoL improvement, happy to take it out if you care strongly.